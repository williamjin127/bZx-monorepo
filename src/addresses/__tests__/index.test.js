/* globals test, expect, describe */
import B0xJS from "../../index";

describe("addresses", () => {
  test("should return an object of addresses", async () => {
    expect(B0xJS.addresses).toMatchSnapshot();
  });
});
