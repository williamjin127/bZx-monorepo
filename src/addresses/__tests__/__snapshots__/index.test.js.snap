// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addresses should return kovan testnet addresses for networkId 42 1`] = `
Object {
  "B0x": "0x246eda094dde8d1f3b5c934d5af0b976e6b3e4a7",
  "B0xOracle": "0xc7c43b7739c4b2708f016ceb2012895ccc55a5db",
  "B0xTo0x": "0x0011bcfaecba1e01d86a8e7d66aef15ee269dd91",
  "B0xToken": "0xe8b6a7fa1976ba6c2d3dd81f063eb25d521186bb",
  "B0xVault": "0x23774c5decb8aa70c0dab82932d6d4cf319e797c",
  "EIP20": "",
  "OracleInterface": "",
  "OracleRegistry": "0x4c3c0326dbee5b9c2f4e9932b0619339894caa70",
  "TokenRegistry": "0x864ea0611b3c3de2178dbef00c9ed8a255c91095",
  "TokenTransferProxy": "0x087eed4bc1ee3de49befbd66c662b434b15d49d4",
  "WETH": "0xd0a1e359811322d97991e03f863a0c30c2cf029c",
  "ZRXToken": "0x6ff6c0ff1d68b964901f986d4c9fa3ac68346570",
}
`;

exports[`addresses should return local testnet addresses for all other networkIds 1`] = `
Object {
  "B0x": "0x8726c7414ac023d23348326b47af3205185fd035",
  "B0xOracle": "0x371b13d97f4bf77d724e78c16b7dc74099f40e84",
  "B0xTo0x": "0x32eecaf51dfea9618e9bc94e9fbfddb1bbdcba15",
  "B0xToken": "0x04b5dadd2c0d6a261bfafbc964e0cac48585def3",
  "B0xVault": "0xb7c9b454221e26880eb9c3101b3295ca7d8279ef",
  "EIP20": "",
  "OracleInterface": "",
  "OracleRegistry": "0x6000eca38b8b5bba64986182fe2a69c57f6b5414",
  "TestToken0": "0xf96b018e8de3a229dbaced8439df9e3034e263c1",
  "TestToken1": "0xd8ca39d276c05a370174ce872b7278fcf734afac",
  "TestToken2": "0x609acc8b356894a937fc58f3411f9528de96ecb1",
  "TestToken3": "0xca9717a4a6e8009b3518648c9f3e7676255471a1",
  "TestToken4": "0x4586649629f699f9a4b61d0e962dc3c9025fe488",
  "TestToken5": "0x5d3ad3561a1235273cbcb4e82fce63a0073d19be",
  "TestToken6": "0xb48e1b16829c7f5bd62b76cb878a6bb1c4625d7a",
  "TestToken7": "0xc4cc602a7345518d0b7a84049d4bc8575ebf3398",
  "TestToken8": "0xe704967449b57b2382b7fa482718748c13c63190",
  "TestToken9": "0xa4b3e1659c473623287b2cc13b194705cd792525",
  "TokenRegistry": "0xc4df27466183c0fe2a5924d6ea56e334deff146a",
}
`;

exports[`addresses should return rinkeby testnet addresses for networkId 4 1`] = `
Object {
  "B0x": "0x66c529139ea5ff2f4e758c28fed902ef78ddf798",
  "B0xOracle": "0x2f498a9cd5f8a74ed579a95a5ab47f1bf9126bfa",
  "B0xTo0x": "0x3cb5ee86adcf8fc4301354b58fde01535939a3b5",
  "B0xToken": "0x14823db576c11e4a54ca9e01ca0b28b18d3d1187",
  "B0xVault": "0xbe46529e2f3208d35027c809ff1df173b022565d",
  "EIP20": "",
  "OracleInterace": "",
  "OracleRegistry": "0x81558e6edf0f7ae1222fd56ad63cffad09dadf64",
  "TokenRegistry": "0x0dadad008647f0f7b68ffe662135cd714a56698c",
  "TokenTransferProxy": "0x4e9aad8184de8833365fea970cd9149372fdf1e6",
  "WETH": "0xc778417e063141139fce010982780140aa0cd5ab",
  "ZRXToken": "0xa8e9fa8f91e5ae138c74648c9c304f1c75003a8d",
}
`;
